File: C/1015.c
#include<stdio.h>
#include<math.h>

int main(void)
{
    double x1,y1,x2,y2,p1,p2,distancia;
    scanf("%lf %lf",&x1,&y1);
    scanf("%lf %lf",&x2,&y2);
    p1=x2-x1;
    p2=y2-y1;
    distancia=sqrt((p1*p1)+(p2*p2));
    printf("%.4lf\n",distancia);
}

File: Kotlin/1015.kt
import kotlin.math.sqrt

fun main(args: Array<String>) {
  val input = Scanner(System.`in`)
  var x1 = input.nextDouble()
  var y1 = input.nextDouble()
  var x2 = input.nextDouble()
  var y2 = input.nextDouble()
  var distancia= sqrt((x2-x1)*(x2-x1) + (y2-y1)*(y2-y1))
  println("%.4f".format(distancia))
}